### DISCUSSION TITLE
Job logs never load

### DISCUSSION THREAD

**Discussion by ntucker** [renovatebot/renovate#21272] [Request Help]
### I'm using the Mend Renovate hosted app on github.com.

- [X] I'm using the Mend Renovate hosted app.

### Was this something which used to work for you, and then stopped?

I never saw this working

### Describe your problem.

People here keep asking for logs, but when I go to https://app.renovatebot.com/dashboard#github/data-client/rest-hooks I just get loading spinners.

### Relevant debug logs.

<details><summary>Logs</summary>

js console

```
Unauthorized [dashboard.js:71:15](https://app.renovatebot.com/js/dashboard.js)
    github https://app.renovatebot.com/js/dashboard.js:71
    jQuery 2
Uncaught (in promise) Unauthorized
Source map error: Error: request failed with status 404
Resource URL: https://app.renovatebot.com/css/bootstrap.min.css
Source Map URL: bootstrap.min.css.map
```

401 network tab:
```
GET /user/installations?per_page=100&page=1 HTTP/2
Host: api.github.com
User-Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:109.0) Gecko/20100101 Firefox/111.0
Accept: application/vnd.github.machine-man-preview+json
Accept-Language: en-US,en;q=0.5
Accept-Encoding: gzip, deflate, br
Referer: https://app.renovatebot.com/
Authorization: token **redacted**
Origin: https://app.renovatebot.com
Connection: keep-alive
Sec-Fetch-Dest: empty
Sec-Fetch-Mode: cors
Sec-Fetch-Site: cross-site
TE: trailers

returns:

{
  "message": "Bad credentials",
  "documentation_url": "https://docs.github.com/rest"
}

```

</details>

**Comment by rarkins** [comment]
I've just tested it in two different accounts (one of which was already logged in, one which wasn't) with two different browsers, and both worked.

Go to https://app.renovatebot.com and either log out or clear all cookies if you can't get that option. Then try logging in again.
